src/BenchmarkDotNet/Reports/Summary.cs

```diff
@@ -73,7 +73,8 @@ public class Summary
 
         private static BenchmarkCase GetFastestBenchmarkCase(ImmutableArray<BenchmarkReport> reports)
         {
-            if (reports.Any() && reports.All(r => r.BenchmarkCase.Config.AutomaticBaselineMode == AutomaticBaselineMode.Fastest))
+            var baselineReport = reports.Where(r => r.BenchmarkCase.Config.AutomaticBaselineMode != AutomaticBaselineMode.None).LastOrDefault();
+            if (baselineReport != null && baselineReport.BenchmarkCase.Config.AutomaticBaselineMode == AutomaticBaselineMode.Fastest)
```

Comment:
```suggestion
            if (reports.Any(r => r.BenchmarkCase.Config.AutomaticBaselineMode == AutomaticBaselineMode.Fastest)
```

You can simplify it.

Reply:
Yes, I just wanted to highlight the idea that we find the latest value and then use it. Since Fastest is the only mode for now, then you suggestion makes sense for sure.